### PR TITLE
Fixes #1215 - Add option to configure text on survey button in Django…

### LINKED ISSUE #1215: Add option to configure text on survey button in Django admin
### Describe your problem or feature you'd like added
The text that displays on the survey button should be configurable by a MyLA administrator. 

### Describe the solution you'd like
Add a page to the Django admin screens that allows admins to change the text that displays on the survey button.
Default to use the text "Take Survey" if no text is specified.

### PR DESCRIPTION
**Pull request by jonespm**
… admin

### CODE REVIEW

**Comment by pushyamig** [comment]
This change is working as expected, even with longer survey text it is expanding nicely and picks up defaults if no text is chosen

Screen Shot 2021-02-11 at 8 44 27 AM

**Review by pushyamig — APPROVED** [review]

**Comment by jonespm** [comment]
I'll review this on mobile and see what how this looks with longer lengths. I'll put a not in the description for a suggested maximum length.

File: dashboard/settings.py [hunk 1]
```diff
@@ -512,7 +512,8 @@
 CONSTANCE_BACKEND = 'constance.backends.database.DatabaseBackend'
 
 CONSTANCE_CONFIG = {
-    'SURVEY_URL': ('', 'URL To qualtrics survey', str),
+    'SURVEY_URL': ('', 'Full URL to Qualtrics survey. If left blank no survey link will display.', str),
+    'SURVEY_TEXT': ('Take Survey', 'Custom text for Qualtrics survey link and title. If left blank will default to "Take Survey". Must also configure SURVEY_URL.', str)
```
Comment (jonespm) [inline dashboard/settings.py:516]:
```suggestion
    'SURVEY_TEXT': ('Take Survey', 'Custom text for Qualtrics survey link and title. If left blank will default to "Take Survey". Must also configure SURVEY_URL. For best mobile fit keep this text to under 6 words/30 characters.', str)
```

Comment (jonespm) [reply dashboard/settings.py:516]: Updated information about best length for text.

**Comment by jonespm** [comment]
Okay I updated the text that will appear in the admin UI. It seems to be okay up to about 30 characters. If we're going to need to go longer than that we'll need a redesign of the top bar in mobile. I think we might want to do this anyway

**Pull request merged**